topeft/modules/corrections.py

```diff
@@ -294,13 +319,12 @@
     for syst in ['','_up','_down','_be1','_be2','_pt1','_pt2']:
         extLepSF.add_weight_sets([("MuonFR_{year}{syst} FR_mva085_mu_data_comb_recorrected{syst} %s" % topcoffea_path(basepathFromTTH + 'fakerate/fr_{year}_recorrected.root')).format(year=year,syst=syst)])
         extLepSF.add_weight_sets([("ElecFR_{year}{syst} FR_mva090_el_data_comb_NC_recorrected{syst} %s" % topcoffea_path(basepathFromTTH + 'fakerate/fr_{year}_recorrected.root')).format(year=year,syst=syst)])
-
 extLepSF.finalize()
 SFevaluator = extLepSF.make_evaluator()
 
 ffSysts=['','_up','_down','_be1','_be2','_pt1','_pt2']
 
-def ApplyTES(year, Taus, isData):
+def ApplyTES(year, Taus, isData, tagger, syst_name, vsJetWP):
```

Comment:
Not exactly relevant to the changes you made here, but could we change `Taus` to `taus`? Also, can we add a comment at least briefly describing the purpose of this function ("TES" is pretty ambiguous)?